js/index.js

```diff
@@ -39,4 +39,46 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+logo.src = 'img/logo.png';
+
+const nav = document.querySelectorAll('nav');
```

Comment:
```suggestion
const navLinks = document.querySelectorAll("nav a");
```
This should solve your issue of selecting a tags